standards/technical-approach.md

```diff
@@ -0,0 +1,49 @@
+# Technical Approach Documents
+
+Technical Approach Documents (TADs) serve to help the engineering team document solutions to feature development, share ideas between teams and help highlight potential issues with tasks before they are encountered during the development process.
+
+TADs should strive to describe the whats, hows and whys of the technical problem that is trying to be solved. It should strive to understand the problem at hand in a comprehensive way and explore any challenges so that engineers can be confident in their implementations of solutions. In short writing a TAD should answer any technical questions prior to the start of actual development work, freeing up developers to focus on implementation once the major questions have been resolved.
+
+## Authors
+
+TADs should be written by the most senior engineer who is or will be working on a project/feature/fix. If it is unknown who will be doing the implementation work for a project an engineer with previous experience or subject matter knowledge should be assigned the task of writing a TAD.
+
+## When Necessary
+
+TADs do not have to be created for all development work, but MUST be created in the following cases:
+
+- If a Business Requirements Document (BRD) has been created
+- If the scope of work is expected to extend beyond a single sprint
+- If multiple developers will be contributing to the new feature
+- If the pros and cons of introducing a new tool or technology to a project, team, or the department need to be considered
+
+TADs SHOULD be created in the following situations:
+
+- If multiple approaches to a feature or problem are valid and the engineer assigned the ticket(s) wishes to document the decisions made
+- If questions are raised about the appropriateness of an approach and engineer/team wishes to document the decision process 
+
+## Process
+
+All TADs MUST follow this process to be approved, at which point JIRA tickets can be created and development work can start.
+
+1. The most senior developer on the feature/task/story is assigned a ticket to write the TAD and the ticket MUST be tagged with `tech-approach`
+2. The assigned developer should collect as much information from the BRD, stakeholders and any other relevant information sources and write the TAD
+3. The TAD MAY be circulated at this point for feedback within a team or stakeholders for further input/feedback
+4. When ready for review the Engineering Leadership Team (ELT) should be notified and the TAD uploaded to the shared TAD Google Drive (see below)
+5. The TAD MUST be reviewed by at least one ELT member with subject area knowledge, and SHOULD be reviewed by two members (see review process below)
+6. If approved in Step 5 the TAD should be sent to Garvita Kapur for final approval
+7. If approved specific tickets should be created to execute the work as described in the TAD
```

Comment:
Can you add how a developer will know that their TAD has been approved? Is it if the JIRA ticket is moved to "Done" or if they hear back from Garvita or another ELT member that it was approved? Is something written in the TAD that says, "Approach X approved for work on [date]"?